raise ImportError(
    "Random number generation has moved to `pystencils.random` in pystencils 2.0."
)
